tests/unit/replicator_tests.py

```diff
@@ -157,6 +157,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @skip_for_iam
     @flaky(max_runs=3)
     def test_create_replication(self):
```

Comment:
Why are we skipping IAM for replication?

Reply:
You need the user/pass for the account in order to place an auth header in the replication doc for the source db (since replication doesn't yet support IAM).

Reply:
Ok I understand, let's skip tests using _replicate endpoint.